src/hooks/resources/device.ts

```diff
@@ -1,5 +1,6 @@
 import * as Bluebird from 'bluebird';
 import * as _ from 'lodash';
+import * as resinSemver from 'resin-semver';
```

Comment:
I have a PR in https://github.com/balena-io/open-balena-api/pull/291 that'll update this to balena-semver btw